crates/net/network/src/session/active.rs

```diff
@@ -255,6 +256,17 @@ impl<N: NetworkPrimitives> ActiveSession<N> {
             EthMessage::Receipts(resp) => {
                 on_response!(resp, GetReceipts)
             }
+            EthMessage::Other(bytes) => {
```

Comment:
yeah I think we need to store the id byte together with the payload (RawCapMessage) here so we don't lose the context